lib/Serge/Application.pm

```diff
@@ -24,12 +25,19 @@ sub new {
 sub run {
     my ($self) = @_;
 
-    my ($help, $debug);
+    my ($help, $debug, $version);
     my $result = GetOptions(
         'help' => \$help,
         'debug' => \$debug,
+        'version|V' => \$version,
```

Comment:
I don't quite agree with this one, as from my understanding Serge is meant to be used by any CLI users, not only those who are familiar and comfortable with Perl development. As such, the vast majority of CLI users will find case insensitivity in flags and options non-intuitive, and as far as I am aware of this behavior has never been advertised as such (I only discovered this was the case while reading through `Getopt::Long` docs).

At the same time, `-v` is [commonly used](http://catb.org/~esr/writings/taoup/html/ch10s05.html) as a short form for `--verbose`, whereas `-V` for `--version`. In this regards, I am inclined to leave it as just `--version` and be set with that.